js/PulsarFormComponent.js

```diff
@@ -84,9 +84,24 @@ PulsarFormComponent.prototype.initTimePickers = function () {
  */
 PulsarFormComponent.prototype.initDatePickers = function () {
     // Attach basic pikaday to datepicker fields
-    this.$html
-        .find('[data-datepicker=true]')
-        .pikaday({ format: 'DD/MM/YYYY' });
+
+    const datepickers = this.$html.find('[data-datepicker="true"]');
+
+    datepickers.each((index, element) => {
```

Comment:
This change will break any existing datepickers which don't have a format option defined, there should be a default fallback to UK dates when no format is supplied.

This should be addressed while refactoring out the multiple find calls.

Reply:
There is 'data-format': options.format|default('default') in the helper. Isn't it enough?

Reply:
@nvasileiadis That unfortunately presumes developers will only use the helper, and doesn't cater for hardcoded markup